site/admission/index.html

```diff
@@ -11,57 +11,79 @@
 
 <body>
   <nav><a href="">Home</a><a href="/about">About</a></nav>
-  <div class="header-box">
-    <img src="../assets/images/placeholder-image-rectangle.png" alt="placeholder-image-rectangle">
-    <h1>Admission</h1>
-    <p>Jodie McSally Dance Program, Class of 21</p>
+  <div class="banner">
+    <div class="admission-title">
+      <h1 class="title">Admission</h1>
+    </div>
+    <div class="placeholder-image">
+      <img src="../assets/images/placeholder-image-rectangle.png" alt="placeholder-image-rectangle">
+    </div>
+  </div>
+  <div class="Admission-nav"></div>
+  <div class="nav-3">
+    <div class="item-1-3">
```

Comment:
I see you haven't used this class yet. Try and stay away from adding a class to each individual item, like `item-1-3`, `item-2-3`, `item-3-3`, etc. Instead try and just give them all the class `nav-item` or something and style them all using the same class name.